src/jdk.incubator.concurrent/share/classes/jdk/incubator/concurrent/StructuredTaskScope.java

```diff
@@ -1088,21 +1097,21 @@ public Optional<Throwable> exception() {
         }
 
         /**
-         * Throws if a task completed abnormally. If any task completed with an exception
-         * then {@code ExecutionException} is thrown with the exception of the first task
-         * to fail as the {@linkplain Throwable#getCause() cause}.
-         * If no task completed with an exception but cancelled tasks were notified to the
+         * Throws if a subtask completed abnormally. If any subtask completed with an
+         * exception then {@code ExecutionException} is thrown with the exception of the
+         * first subtask to fail as the {@linkplain Throwable#getCause() cause}. If no
+         * subtask completed with an exception but cancelled subtasks were notified to the
          * {@code handleComplete} method then {@code CancellationException} is thrown.
-         * This method does nothing if no tasks completed abnormally.
+         * This method does nothing if no subtasks completed abnormally.
          *
          * <p> This method is intended to be invoked by the task scope owner after it has
          * invoked {@link #join() join} (or {@link #joinUntil(Instant) joinUntil}).
          * The behavior of this method is unspecified when invoking this method before
-         * the join is done.
+         * the {@code join} is invoked.
```

Comment:
```suggestion
         * {@link #join} is invoked.
```

Reply:
More generally, I see that you used `{@code ... }` in a lot of places where `{@link ... }` could also be used. In some of those places (like this one) where there is a clear cross-reference, I think `@link` could be preferrable. The only case where `@code` is fine is when referring to the name of the class itself (e.g. `{@code StructuredTaskScope}`). But of course this is subjective.

Reply:
Also, note the typo `the join is invoked`. Either `the` is dropped, or `method` is added. I've seen more than one occurrence of this.

Reply:
It should be "before join is invoked". It doesn't use a link here because it already links to join and joinUntil in the previous sentence.